cmd/peer-rest-server.go

```diff
@@ -403,6 +403,9 @@ func (s *peerRESTServer) DispatchNetOBDInfoHandler(w http.ResponseWriter, r *htt
 		return
 	}
 
+	done := keepHTTPResponseAlive(w)
+	defer done()
```

Comment:
You must call done before you write to the connection.

Reply:
@wlan0 ^^

Reply:
done()